pkg/reconciliation/reconcile_racks.go

```diff
@@ -1803,7 +1803,13 @@ func (rc *ReconciliationContext) startOneNodePerRack(endpointData httphelper.Cas
 
 	// if the DC has no ready seeds, label a pod as a seed before we start Cassandra on it
 	// and also consider additional seeds
-	labelSeedBeforeStart := readySeeds == 0 && len(rc.Datacenter.Spec.AdditionalSeeds) == 0
+
+	externalSeedPoints := 0
+	if existingEndpoints, err := rc.GetAdditionalSeedEndpoint(); err == nil {
+		externalSeedPoints = len(existingEndpoints.Subsets[0].Addresses)
+	}
+
+	labelSeedBeforeStart := readySeeds == 0 && len(rc.Datacenter.Spec.AdditionalSeeds) == 0 && externalSeedPoints == 0
```

Comment:
Are we sure that we want to check external seeds here? I believe this is where the first seed (or seeds) is labeled and started. Subsequent seed labeling is done in `checkSeedLabels()`. If have an external seed here and have not yet started any C* nodes in this dc, I fear that this might prevent any nodes from starting.